docs/_includes/Running-Alluxio-Fault-Tolerant/jps.md

```diff
@@ -0,0 +1,3 @@
+```bash
```

Comment:
@bf8086 this is the old convention but we are currently moving away from putting code snippets under `docs/_includes`.
Instead, simply inline the code in the original .md file. In this way, it is easier to edit and maintain.

Reply:
Done.